### PR TITLE
Add confirm saving changes before closing

### PR DESCRIPTION
**Pull request by henri**
added command for general -> auto save prompt

### CODE REVIEW

File: defaults.yml [hunk 1]
```diff
@@ -1053,4 +1073,4 @@ categories:
               width: 696
               height: 252
             text: Repeats the key as long as it is held down.
-        versions: [Monterey, Big Sur]
+        versions: [Monterey, Big Sur]z
```
Comment (Gudsfile) [inline defaults.yml:1076]:
```suggestion
        versions: [Monterey, Big Sur]
```
Oops, it must be a mistake.

File: defaults.yml [hunk 2]
```diff
@@ -962,6 +962,26 @@ categories:
...
+        description:
+          Should you be asked to keep changes when closing documents or just have changes saved automatically?
+        param:
+          type: int
+        examples:
+          - value: 0
+            default: 0
+            text: Auto saving is enabled, you are not prompted to save changes.
+          - value: 2
+            text: Auto saving is disabled, you are prompted if you want to save changes.
```
Comment (Gudsfile) [inline defaults.yml:982]: Why not use a boolean? I think it's more clear with "true" or "false".

Maybe @yannbertrand  has a preference?

**Review by Gudsfile — CHANGES_REQUESTED** [review]

File: defaults.yml (hunk 1 above)
Comment (henri) [reply defaults.yml:1076]: Sounds good. I am guessing it will work on Big Sur as well. I have not tested.

File: defaults.yml (hunk 2 above)
Comment (henri) [reply defaults.yml:982]: For this value, I am actually not sure if boolean will work. Would require some testing. I agree true or false is far more clear. But I would have guessed that would use values 0 and 1. Not sure why but when you tick it in the system preferences it sets to integer 2.

Comment (henri) [reply defaults.yml:982]: Perhaps there is a third state between these. I have not investigated any further.

File: defaults.yml (hunk 1 above)
Comment (yannbertrand) [reply defaults.yml:1076]: Actually you forgot a `z` there 😄

**Comment by yannbertrand** [comment]
Couldn't merge due to a conflict. I manually pushed it in 126524a. Thanks!

**Pull request closed without merging**